npm/builder.js

```diff
@@ -465,3 +467,28 @@ Builder.prototype.compile = function() {
   asciidoctorCSSFile = asciidoctorGemPath + '/data/stylesheets/asciidoctor-default.css';
   fs.createReadStream(asciidoctorCSSFile).pipe(fs.createWriteStream('build/asciidoctor.css'));
 }
+
+Builder.prototype.benchmark = function(runner, callback) {
+  var builder = this;
+
+  async.series([
+    function(callback) { builder.build(callback); },
+    function(callback) {
+      builder.mkdirSync(builder.benchmarkBuildDir);
+      builder.copyToDir('benchmark/run.js', builder.benchmarkBuildDir);
+      callback();
+    },
+    function(callback) {
+      log.title('fetch content from AsciiDoc repository');
+      callback();
+    },
+    function(callback) { builder.fetchAsciiDocContent('asciidoc.txt', 'build/benchmark/userguide.adoc', callback); },
+    function(callback) { builder.fetchAsciiDocContent('customers.csv', 'build/benchmark/customers.csv', callback); },
```

Comment:
I named the method `fetchAsciiDocContent` because it fetch content *from* the AsciiDoc repository :confounded: 

 `fetchContentFromAsciiDocRepository` ?